tests/test_prompts.py

```diff
@@ -15,3 +15,11 @@ def test_prompt_style_from_name():
 
     for style_name in prompt_styles:
         assert isinstance(PromptStyle.from_name(style_name), prompt_styles[style_name])
+
+
+def test_prompt_style_from_config():
```

Comment:
Can you add a test where we go over all the PromptStyle's and call `apply` on them? You can assert that the input prompt is in the output